app/views/trainees/index.html.erb

```diff
@@ -10,6 +10,19 @@
 
 <%= render Trainees::Filters::View.new(@filters) do %>
   <% if @trainees.any? %>
```

Comment:
If all of your trainees are in a draft state, this `if @trainees.any?` will mean that you don't see any of them :(